src/System.Runtime.Analyzers/UnitTests/NormalizeStringsToUppercaseTests.cs

```diff
@@ -9,12 +11,332 @@ public class NormalizeStringsToUppercaseTests : DiagnosticAnalyzerTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicNormalizeStringsToUppercaseAnalyzer();
+            return new NormalizeStringsToUppercaseAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpNormalizeStringsToUppercaseAnalyzer();
+            return new NormalizeStringsToUppercaseAnalyzer();
         }
+
+        #region No Diagnostic Tests
+
+        [Fact]
+        public void NoDiagnostic_ToUpperCases()
+        {
+            VerifyCSharp(@"
+using System;
+using System.Globalization;
+
+public class UnexpectedWarningAttribute : Attribute
```

Comment:
(Also, that "FxCop tests" like points to an internal site, so you should probably remove it...)